src/java/org/apache/cassandra/security/AbstractSslContextFactory.java

```diff
@@ -263,4 +264,11 @@ protected SslProvider getSslProvider()
     abstract protected KeyManagerFactory buildKeyManagerFactory() throws SSLException;
 
     abstract protected TrustManagerFactory buildTrustManagerFactory() throws SSLException;
+
+    /**
+     * Internode mTLS needs separate keystore for outbound connections.
+     * @return
+     * @throws SSLException
+     */
```

Comment:
Consider the following.
```suggestion
    /**
     * Create a {@code KeyManagerFactory} for outbound connections. 
     * It provides a seperate keystore for internode mTLS connections.
     * @return {@code KeyManagerFactory}
     * @throws SSLException
     */
```